tutorials/plot_stats_spatio_temporal_cluster_sensors.py

```diff
@@ -191,6 +183,10 @@
 # ----------
 #
 # - What is the smallest p-value you can obtain, given the finite number of
-#    permutations?
+#   permutations?
 # - use an F distribution to compute the threshold by traditional significance
-#    levels. Hint: take a look at ``scipy.stats.distributions.f``
+#   levels. Hint: take a look at :class:`scipy.stats.distributions.f`
```

Comment:
Inspecting the HTML of SciPy suggests told me that it might just be:
```
:obj:`scipy.stats.f`
```
and using `make html_dev-noplot` locally suggests it should work.